data/scripts.js

```diff
@@ -1715,29 +1715,31 @@ exports.BattleScripts = {
 			item = 'Rocky Helmet';
 		} else if (counter.Physical + counter.Special >= 4 && template.baseStats.def + template.baseStats.spd > 189) {
 			item = 'Assault Vest';
-		} else if (counter.Physical + counter.Special >= 4) {
+		} else if (counter.Physical + counter.Special >= 4 && ability === 'Multiscale') {
 			item = (hasMove['fakeout'] || hasMove['return'] || hasMove['extremespeed'] || (hasMove['suckerpunch'] && !hasType['Dark'])) ? 'Life Orb' : 'Expert Belt';
 		} else if (setupType && hasMove['outrage']) {
 			item = 'Lum Berry';
 		} else if (counter.Physical + counter.Special >= 3 && !!counter['speedsetup'] && template.baseStats.hp + template.baseStats.def + template.baseStats.spd > 300) {
 			item = 'Weakness Policy';
-		} else if ((counter.Physical + counter.Special >= 3) && ability !== 'Sturdy' && !hasMove['dragontail']) {
+		} else if ((counter.Physical + counter.Special >= 3) && ability !== 'Sturdy' && ability !== 'Multiscale' && !hasMove['dragontail']) {
```

Comment:
This is unneeded now because Dragonite always has Outrage because it is STAB. So, if it has four attacks then it gets Choice and if it doesn't then it gets Lum Berry from above. Lugia will never get Life Orb because it doesn't have that many attacking moves.